test/check-machines-create

```diff
@@ -85,7 +85,7 @@ class TestMachinesCreate(machineslib.VirtualMachinesCase):
                                                             location=config.TREE_URL,
                                                             memory_size=128, memory_size_unit='MiB',
                                                             storage_pool=NO_STORAGE,
-                                                            os_name=config.CENTOS_7,
+                                                            os_name=config.RHEL_8_10,
```

Comment:
This commit derivates from `main`. I think we should also drop CENTOS_7 in `main`